features/step_definitions/opportunities_steps.rb

```diff
@@ -20,16 +20,17 @@
   page.should have_selector(:css, ".search-result")
 end
 
-Then (/^I see that the stated number of results does not exceed that (\w+)$/) do |variable_name|
-  var = instance_variable_get("@#{variable_name}")
+Then (/^I see that the stated number of results (does not exceed|equals|is no fewer than) that (\w+)$/) do |comparison_string, variable_name|
+  var_val = instance_variable_get("@#{variable_name}")
   @new_result_count = page.first(:css, ".search-summary-count").text.to_i
   puts "Number of results: #{@new_result_count}"
-  @new_result_count.should <= var
-end
-
-Then (/^I see that the stated number of results equals that (\w+)$/) do |variable_name|
-  var = instance_variable_get("@#{variable_name}")
-  var.should == page.first(:css, ".search-summary-count").text.to_i
+  if comparison_string == "does not exceed"
+    @new_result_count.should <= var_val
+  elsif comparison_string == "equals"
+    @new_result_count.should == var_val
+  elsif comparison_string == "is no fewer than"
```

Comment:
I'd have preferred "is at least" but you might want to canvass others to see what's more readable. I mean, I find `>=` more readable than text altogether, but I suppose that's not really the idea :-)

Reply:
I agree if you look at it on its own but if you look at it in the sentence, it doesn't seem to fit as well, especially given that the text following it can be any valid `MAYBE_VAR` entry...